frontend/src/components/PostAddingForm/PostAddingForm.jsx

```diff
@@ -14,10 +16,16 @@ import {
   Button,
   LeftColumn,
   RightColumn,
+  ErrorText,
 } from './PostAddingForm.styled';
 
 const PostAddingForm = ({ formSubmit }) => {
```

Comment:
Nieużywany parametr, można wywalić.
```suggestion
const PostAddingForm = () => {
```